R/fb_ad.R

```diff
@@ -183,7 +183,7 @@ fbad_list_ad <- function(fbacc, id, statuses, fields = 'id') {
     }
 
     ## default ID for current Ad Account
-    if (missing(id) | fn == 'fbad_list_campaign') {
+    if (missing(id) | fn %in% c('fbad_list_campaign', 'fbad_list_audience')) {
```

Comment:
Are you sure this extra check is required? I think the `missing(id)` already covers that as we are/should not set and `id` when running `fbad_list_audience`